### ISSUE TITLE
Show tags

### ISSUE THREAD

**Issue by johanhelsing** [TomasTomecek/git-stats#40]
It would be nice if it was possible to see what tag(s) I had checked out, i.e. when doing:

    git checkout v1.0.0

I would like to see v1.0.0 somewhere in the prompt.

**Comment by TomasTomecek** [comment]
This is a good suggestion. Sadly, this is a limitation of git: tags are not branches, so git treats them as a detached state. I feel like this could be doable, it would just require some tinkering with libgit.

**Comment by johanhelsing** [comment]
I think fish's built in git prompt does this, maybe look there for how they do it?

**Comment by johanhelsing** [comment]
Fish seems to use: `git describe --tags --exact-match HEAD`

**Comment by TomasTomecek** [comment]
Yup, that seems to work. The problem is that this tool is using libgit, so I would need to figure that out using the rust binding.

Right now p-g-p has like 5 methods to deal with branches and refs:

https://github.com/TomasTomecek/pretty-git-prompt/blob/master/src/backend.rs#L129
